definitions/divorce/json/CaseEventToFields/CaseEventToFields-prod.json

```diff
@@ -0,0 +1,337 @@
+    "PageDisplayOrder": 1,
+    "PageColumnNumber": 1,
+    "ShowSummaryChangeOption": "Yes"
+  },
+  {
+    "LiveFrom": "20/09/2019",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "EnterAOSOffline_Respondent",
+    "CaseFieldID": "D8RespondentHomeAddress",
+    "PageFieldDisplayOrder": 15,
+    "DisplayContext": "OPTIONAL",
+    "PageID": "aosdataentry",
+    "PageLabel": "Enter scanned form case data",
+    "PageDisplayOrder": 1,
+    "PageColumnNumber": 1,
+    "ShowSummaryChangeOption": "Yes"
+  },
+  {
+    "LiveFrom": "20/09/2019",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "EnterAOSOffline_Respondent",
+    "CaseFieldID": "RespEmailAddress",
+    "PageFieldDisplayOrder": 17,
+    "DisplayContext": "OPTIONAL",
+    "PageID": "aosdataentry",
+    "PageLabel": "Enter scanned form case data",
+    "PageDisplayOrder": 1,
+    "PageColumnNumber": 1,
+    "ShowSummaryChangeOption": "Yes"
+  },
+  {
+    "LiveFrom": "20/09/2019",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "EnterAOSOffline_Respondent",
+    "CaseFieldID": "solAOSSOT",
+    "PageFieldDisplayOrder": 18,
+    "DisplayContext": "MANDATORY",
+    "PageID": "aosdataentry",
+    "PageLabel": "Enter scanned form case data",
+    "PageDisplayOrder": 1,
+    "PageColumnNumber": 1,
+    "ShowSummaryChangeOption": "Yes"
+  },
+  {
+    "LiveFrom": "20/09/2019",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "EnterAOSOffline_Respondent",
+    "CaseFieldID": "D8RespondentCorrespondenceAddress",
+    "PageFieldDisplayOrder": 19,
+    "DisplayContext": "OPTIONAL",
+    "PageID": "aosdataentry",
+    "PageLabel": "Enter scanned form case data",
+    "PageDisplayOrder": 1,
+    "PageColumnNumber": 1,
+    "ShowSummaryChangeOption": "Yes"
+  },
+  {
+    "LiveFrom": "20/09/2019",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "EnterAOSOffline_Respondent",
+    "CaseFieldID": "D8RespondentSolicitorName",
+    "PageFieldDisplayOrder": 20,
+    "DisplayContext": "OPTIONAL",
+    "PageID": "aosdataentry",
+    "PageLabel": "Enter scanned form case data",
+    "PageDisplayOrder": 1,
+    "PageColumnNumber": 1,
+    "ShowSummaryChangeOption": "Yes"
+  },
+  {
+    "LiveFrom": "20/09/2019",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "EnterAOSOffline_Respondent",
+    "CaseFieldID": "D8RespondentSolicitorEmail",
+    "PageFieldDisplayOrder": 21,
+    "DisplayContext": "OPTIONAL",
+    "PageID": "aosdataentry",
+    "PageLabel": "Enter scanned form case data",
+    "PageDisplayOrder": 1,
+    "PageColumnNumber": 1,
+    "ShowSummaryChangeOption": "Yes"
+  },
+  {
+    "LiveFrom": "20/09/2019",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "EnterAOSOffline_Respondent",
+    "CaseFieldID": "D8RespondentSolicitorPhone",
+    "PageFieldDisplayOrder": 22,
+    "DisplayContext": "OPTIONAL",
+    "PageID": "aosdataentry",
+    "PageLabel": "Enter scanned form case data",
+    "PageDisplayOrder": 1,
+    "PageColumnNumber": 1,
+    "ShowSummaryChangeOption": "Yes"
+  }
+]
```

Comment:
Moved previous CaseEventsToFields for AOS offline Respondent here. This prevents duplicates and build passes and also allows the current prod state to remain and make clear which fields needs updating when it does to prod.

Reply:
Technically, those fields are not available on prod yet, so it might not be necessary to have this file.